src/main/java/fi/aalto/cs/apluscourses/presentation/base/Searchable.java

```diff
@@ -0,0 +1,10 @@
+package fi.aalto.cs.apluscourses.presentation.base;
+
+import org.jetbrains.annotations.NotNull;
+
+public interface Searchable {
+  @NotNull
+  default String getSearchableString() {
+    return "";
```

Comment:
Nit: `StringUtils.EMPTY` or externalizing the `""` into props, what do you think :question: